### ISSUE TITLE
`histplot` Legend Appearing Twice

### ISSUE THREAD

**Issue by jasonhadiputra** [mwaskom/seaborn#3172]
![image](https://user-images.githubusercontent.com/32168189/205221286-8660398c-9e9e-434e-97f0-7266b7969d31.png)

It seems that the histogram legend appears twice. I expected it to show only once. Did I miss something?

**Comment by mwaskom** [comment]
Duplicate of https://github.com/mwaskom/seaborn/issues/3115
